docker/service/src/main/java/uk/nhs/adaptors/scr/models/GpSummary.java

```diff
@@ -71,6 +72,7 @@ public class GpSummary {
     private List<RiskToPatient> risksToPatient = new ArrayList<>();
     private List<Treatment> treatments = new ArrayList<>();
     private List<SocialOrPersonalCircumstance> socialOrPersonalCircumstances = new ArrayList<>();
+    private List<FamilyHistory> familyHistories = new ArrayList<>();
```

Comment:
to make this useful.... need to add the family histories stuff to the observation mapper. we can do this in another PR and mop up some of the other bits over next couple of days